src/handler/global/personalstats.ts

```diff
@@ -147,6 +149,9 @@ function getPoints(user: PrivateUserData): number {
         const n = user.skills[skill as keyof Skills];
         points += (n * (n + 1)) / 2;
     }
+    if (user.unlockedUnits?.length !== 0) {
```

Comment:
linter warning here

Reply:
for some reason running the linter isnt working locallym and its passing the build here... did you break it?

Reply:
works just fine for me

Reply:
just remove the unnecessary optional chaining >.>